x-pack/plugins/security_solution/cypress/integration/network/overflow_items.spec.ts

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  ADD_TO_TIMELINE,
+  COPY,
+  EXPAND_OVERFLOW_ITEMS,
+  FILTER_IN,
+  FILTER_OUT,
+  SHOW_TOP_FIELD,
+} from '../../screens/network/flows';
+import { cleanKibana } from '../../tasks/common';
+
+import { loginAndWaitForPage } from '../../tasks/login';
+
+import { NETWORK_URL } from '../../urls/navigation';
+
+const testDomainOne = 'endpoint-dev-es.app.elstc.co';
+const testDomainTwo = 'endpoint2-dev-es.app.elstc.co';
+
+describe('Overfow items', () => {
+  context('Network stats and tables', () => {
+    beforeEach(() => {
+      cleanKibana();
```

Comment:
cleanKibana removes and adds again all the kibana indexes, so this method slows down the execution of the tests. Taking into account that these tests do not modify any kibana index, there is no need of executing it between tests. Just doing it one time is enough. So it is better to extract it to a before hook.